plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -138,6 +139,15 @@ func findNodesThatFit(pod *api.Pod, podLister algorithm.PodLister, predicateFunc
 			filtered = append(filtered, node)
 		}
 	}
+	if len(filtered) > 0 && len(extenders) != 0 {
+		for _, extender := range extenders {
+			filteredList, err := extender.Filter(pod, &api.NodeList{Items: filtered})
```

Comment:
I feel the opposite, when you give someone such a simple interface they are going to use it, rather than 'doing it right'...